templates/managed-rds-postgresql-v1.j2

```diff
@@ -0,0 +1,284 @@
+              owner: root
+              group: root
+        /lib/systemd/system/cfn-hup.service:
+          content: !Sub |
+            [Unit]
+        Description=cfn-hup daemon
+
+        [Service]
+        Type=simple
+        ExecStart=/opt/aws/bin/cfn-hup
+        Restart=always
+
+        [Install]
+        WantedBy=multi-user.target
+        mode: "000400"
+        owner: root
+        group: root
+    commands:
+      01_enable_cfn-hup:
+        command: "/bin/systemctl enable cfn-hup.service"
+      02_start_cfn-hup:
+        command: "/bin/systemctl start cfn-hup.service"
+        tag_volume:
+      commands:
+        01_tag_root_disk:
+          command: !Join
+            - ''
+            - - "DB_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) "
+              - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
+          - "--filters Name=attachment.instance-id,Values=$DB_INSTANCE_ID "
+          - "--query Volumes[].VolumeId --out text); "
+          - "/usr/bin/aws ec2 create-tags --region $AWS_REGION --resources $ROOT_DISK_ID "
+          - "--tags Key=Name,Value=$DB_INSTANCE_ID-root Key=cloudformation:stack-name,Value=$STACK_NAME "
+          - "Key=Department,Value=$DEPARTMENT Key=Project,Value=$PROJECT Key=OwnerEmail,Value=$OWNER_EMAIL"
+          env:
+            AWS_REGION: !Ref AWS::Region
+            STACK_NAME: !Ref AWS::StackName
+            DEPARTMENT: !Ref Department
+            PROJECT: !Ref Project
+            OWNER_EMAIL: !Ref OwnerEmail
+    Properties:
+      AllocatedStorage: !Ref 'VolumeSize'
+      DBInstanceClass: !Ref 'InstanceType'
+      KmsKeyId: !If [HasKmsKeyId, !Ref KmsKeyId, !Ref 'AWS::NoValue']
+      StorageEncryption: !Ref 'EncryptVolume'
+      Engine: postgres
+      EngineVersion: !Ref 'DBEngineVersion'
+      MasterUsername: !Ref 'DBMasterUsername'
+      MasterUserPassword: !Ref 'DBMasterUserPassword'
+      MultiAZ: !Ref 'MultiAZ'
+      BackupRetentionPeriod: !Ref 'DBBackupRetentionPeriod'
+      Tags:
+        - Key: Name
+          Value: PostgreSQL Master Database
+      VPCSecurityGroups:
+        - !GetAtt 'DBEC2SecurityGroup.GroupId'
+    DeletionPolicy: Snapshot
+    Tags:
+      - Key: "Name"
+        Value: !Ref 'AWS::StackName'
+      - Key: "Department"
+        Value: !Ref Department
+      - Key: "Project"
+        Value: Project
+      - Key: "OwnerEmail"
+        Value: !Ref OwnerEmail
+Outputs:
+  DBInstance:
+    Value: !Ref DBInstance
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-DBInstance'
+  DBInstancePublicIp:
+    Condition: PublicDBResource
+    Value: !GetAtt 'DBInstance.Endpoint.Address'
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-DBInstancePublicIp'
+  DBInstancePrivateIp:
+    Condition: PrivateDBResource
+    Value: !GetAtt 'DBInstance.Endpoint.Address'
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-DBInstancePrivateIp'
+  OwnerEmail:
```

Comment:
You probably don't need this.  it's only used to by the sceptre notification hook.   i think you are going to notify those dudes yourself.

Reply:
I'm hoping Andrew or James can request it through the provisioner

Reply:
But, I'm removing those as the JDBC Connection string is sufficient.

Reply:
Removing... I'll resolve this comment.

Reply:
huh? it's not removed?